### PR TITLE
Tf2OpenAPI: Add CLI documentation

### PR DESCRIPTION
**Pull request by jc2729**
**What this PR does / why we need it**: Add CLI documentation to Tf2OpenAPI

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kfserving/220)

### CODE REVIEW

File: tools/tf2openapi/README.md [hunk 1]
```diff
@@ -14,5 +14,23 @@ The outcome of this project is a TensorFlow-to-OpenAPI transformer which takes S
...
-  ## TensorFlow Compatibility
-* This tool is compatible with TensorFlow versions 1.xx up to and including 1.13.1. To make it compatible with future TensorFlow versions, you will need to compile the TensorFlow protos and convert them to the internal models. See [DEVELOPER_GUIDE](DEVELOPER_GUIDE.md) for potential issues and solutions.
\ No newline at end of file
+## TensorFlow Compatibility
+* This tool is compatible with TensorFlow versions 1.xx up to and including 1.13.1. To make it compatible with future TensorFlow versions, you will need to compile the TensorFlow protos and convert them to the internal models. See [DEVELOPER_GUIDE](DEVELOPER_GUIDE.md) for potential issues and solutions.
+
+## Usage
+```
+Usage:
+  tf2oas [flags]
```
Comment (ellis-bigelow) [inline tools/tf2openapi/README.md]: let's make this tf2openapi

File: tools/tf2openapi/README.md [hunk 2]
```diff
@@ -14,5 +14,23 @@ The outcome of this project is a TensorFlow-to-OpenAPI transformer which takes S
...
 
-  ## TensorFlow Compatibility
-* This tool is compatible with TensorFlow versions 1.xx up to and including 1.13.1. To make it compatible with future TensorFlow versions, you will need to compile the TensorFlow protos and convert them to the internal models. See [DEVELOPER_GUIDE](DEVELOPER_GUIDE.md) for potential issues and solutions.
\ No newline at end of file
+## TensorFlow Compatibility
+* This tool is compatible with TensorFlow versions 1.xx up to and including 1.13.1. To make it compatible with future TensorFlow versions, you will need to compile the TensorFlow protos and convert them to the internal models. See [DEVELOPER_GUIDE](DEVELOPER_GUIDE.md) for potential issues and solutions.
+
+## Usage
```
Comment (ellis-bigelow) [inline tools/tf2openapi/README.md]: Let's put this Usage section first. Right before overview

File: tools/tf2openapi/cmd/main.go [hunk 3]
```diff
@@ -1,42 +1,61 @@
...
+			contain signature definitions (SignatureDefs) for models.`,
+		Run: viewAPI,
+	}
+
+	rootCmd.Flags().StringVarP(&model, "model", "m", "", "Absolute path of SavedModel file")
+	rootCmd.MarkFlagRequired("model")
+	rootCmd.Flags().StringVarP(&modelName, "name", "n", "model", "Name of model")
+	rootCmd.Flags().StringVarP(&modelVersion, "version", "v", "1", "Model version")
+	rootCmd.Flags().StringVarP(&outFile, "output-file", "o", "", "Absolute path of file to write OpenAPI spec to")
+	rootCmd.Flags().StringVarP(&sigDefKey, "signature-def", "s", "", "Serving Signature Def Key")
```
Comment (ellis-bigelow) [inline tools/tf2openapi/cmd/main.go]: these are all super cool.

File: tools/tf2openapi/cmd/main.go [hunk 4]
```diff
@@ -1,42 +1,61 @@
 package main
 
 import (
-	"flag"
 	"github.com/golang/protobuf/proto"
-	"github.com/kubeflow/kfserving/tools/tf2openapi/generator"
 	pb "github.com/kubeflow/kfserving/tools/tf2openapi/generated/protobuf"
+	"github.com/kubeflow/kfserving/tools/tf2openapi/generator"
+	"github.com/spf13/cobra"
```
Comment (ellis-bigelow) [inline tools/tf2openapi/cmd/main.go]: Do we already depend on this explicitly? If we don't let's take an explicit dependency.

File: tools/tf2openapi/README.md [hunk 5]
```diff
@@ -14,5 +14,23 @@ The outcome of this project is a TensorFlow-to-OpenAPI transformer which takes S
...
+```
+Usage:
+  tf2oas [flags]
+
+Required Flags:
+  -m, --model string           Absolute path of SavedModel file
+  
+Flags:
+  -h, --help                   help for tf2oas
+  -m, --model string           Absolute path of SavedModel file
```
Comment (ellis-bigelow) [inline tools/tf2openapi/README.md]: --model_base_dir or whatever tensorflow uses?

Comment (jc2729) [reply tools/tf2openapi/README.md]: got it, changing to model_base_path https://www.tensorflow.org/tfx/serving/docker#running_a_serving_image

File: tools/tf2openapi/cmd/main.go (hunk 4 above)
Comment (jc2729) [reply tools/tf2openapi/cmd/main.go]: added to Gopkg.toml & Gopkg.lock

**Comment by ellis-bigelow** [comment]
Awesome work!
/lgtm
/approve

**Pull request merged**